templates/scitas/arvine.yaml.j2

```diff
@@ -43,7 +43,9 @@
   - eigen
   - fastqc
   - ffmpeg
+  - ffmpeg +libx264
   - fftw ~mpi+openmp
+  - freefem
```

Comment:
Are you trying to sneak in an other spec ?

Reply:
You caught me !